test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperFormTest.cs

```diff
@@ -434,50 +433,6 @@ public void BeginForm_EndForm_RendersAntiforgeryTokenWhenMethodIsPost()
             // Act & Assert
             using (var form = htmlHelper.BeginForm(FormMethod.Post, antiforgery: null, htmlAttributes: null))
             {
-                Assert.True(viewContext.FormContext.HasAntiforgeryToken);
-            }
-
-            Assert.Equal(
-                "<form><antiforgery></antiforgery></form>",
-                writer.GetStringBuilder().ToString());
-        }
-
-        // This is an integration for the implicit antiforgery token added by BeginForm.
-        [Fact]
-        public void BeginForm_EndForm_RendersAntiforgeryToken_WithExplicitCallToAntiforgery()
-        {
-            // Arrange
-            var htmlGenerator = new Mock<IHtmlGenerator>(MockBehavior.Strict);
-            htmlGenerator
-                .Setup(g => g.GenerateForm(
-                    It.IsAny<ViewContext>(),
-                    It.IsAny<string>(),
-                    It.IsAny<string>(),
-                    It.IsAny<object>(),
-                    It.IsAny<string>(),
-                    It.IsAny<object>()))
-                .Returns(new TagBuilder("form"));
-
-            htmlGenerator
-                .Setup(g => g.GenerateAntiforgery(It.IsAny<ViewContext>()))
-                .Returns(new TagBuilder("antiforgery"));
-
-            var htmlHelper = DefaultTemplatesUtilities.GetHtmlHelper(htmlGenerator.Object);
-            var serviceProvider = new Mock<IServiceProvider>();
-            serviceProvider.Setup(s => s.GetService(typeof(HtmlEncoder))).Returns(new HtmlTestEncoder());
-            var viewContext = htmlHelper.ViewContext;
-            viewContext.HttpContext.RequestServices = serviceProvider.Object;
-
-            var writer = viewContext.Writer as StringWriter;
-            Assert.NotNull(writer);
-
-            // Act & Assert
-            using (var form = htmlHelper.BeginForm())
-            {
-                Assert.True(viewContext.FormContext.HasAntiforgeryToken);
-
-                // This call will no-op
```

Comment:
Since this is no longer a criteria that needs to be tested from a HtmlHelper perspective and also since its being covered in DefaultHtmlGenerator, we can delete this test.